apps/alphabet_mobile/index.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * @format
+ */
+
+import {AppRegistry} from 'react-native';
```

Comment:
The formatting appears different than in other parts of our codebase. Can this be fixed?